README.md

```diff
@@ -39,9 +39,9 @@ const stringLiteral = "Sorry, your upload failed.";
 const templateString = `We are sorry ${name}, but something went wrong!`;
 ```
 
-### prefer-quot
+### prefer-rsquo
 
-This rule encourages the use of &quot (’) over &apos (') in text. This rule is **auto fixable**.
+This rule encourages the use of the right side quote &rsquo (’) over &apos (') in text. This rule is **auto fixable**.
```

Comment:
I believe the `s` in `rsquo` stands for single, not side.

Reply:
Ah ok, yeah the page I looked at obviously got it wrong!